lib/client-options.js

```diff
@@ -318,4 +323,6 @@ exports.coreConnectionsPerHostV3 = coreConnectionsPerHostV3;
 exports.createQueryOptions = createQueryOptions;
 exports.maxRequestsPerConnectionV2 = maxRequestsPerConnectionV2;
 exports.maxRequestsPerConnectionV3 = maxRequestsPerConnectionV3;
-exports.setProtocolDependentDefaults = setProtocolDependentDefaults;
\ No newline at end of file
+exports.setProtocolDependentDefaults = setProtocolDependentDefaults;
+
+const Host = require('./host').Host;
```

Comment:
delayed require until after `exports.defaultOptions` otherwise we create cyclic dependency with `host` -> `host-connection-pool` -> `defaultOptions`